core/frontend/services/themes/engines/create.js

```diff
@@ -28,8 +28,8 @@ module.exports = (packageJson) => {
         if (packageJson.engines['ghost-api']) {
             const availableApiVersions = {};
             config.get('api:versions:all').forEach((version) => {
-                if (version === 'canary') {
-                    availableApiVersions['canary'] = version;    
+                if (version === 'canary') { // eslint-disable-dot-notation
+                    availableApiVersions['canary'] = version;
```

Comment:
Do we have to `// eslint-disable-dot-notation` here? Doesn't seem necessary tbh :thinking:

Reply:
Hmm seems it got messed up on commit, but was supposed to be for next line. Travis was failing for it suggesting using dot-notation, thus the reason for adding.